internal/pkg/manifest/validate.go

```diff
@@ -843,6 +845,15 @@ func (ip IPNet) validate() error {
 
 // validate returns nil if NetworkLoadBalancerConfiguration is configured correctly.
 func (c NetworkLoadBalancerConfiguration) validate() error {
+	for idx, listener := range c.NLBListeners() {
+		if err := listener.validate(); err != nil {
+			return fmt.Errorf("validate nlb.listener[%d]; %s", idx+1, err)
```

Comment:
Why `idx+1` 🤔 ? Can we validate the main listener first, and then loop through the additional listeners? This way we can just use `idx` when wrapping errors from the additional listeners.

Reply:
Do you mean 1st validate the 0th index separately and then loop over remaining items? 
I kept idx+1 so that customer won't see listener[0] and get confused. Is it right practice to show 0th index to the customer?

Reply:
I meant ⬇️ 
```
if err := c.MainListener; err != nil {
}

for idx, listener := range c.AdditionalListeners {
}
```

Reply:
I think all of our customers are programmers, starting with `0` is probably fine..